app/scripts/modules/canary/canary/canaryStage.js

```diff
@@ -420,8 +420,8 @@ module.exports = angular
         const config = CloudProviderRegistry.getValue(getCloudProvider(), 'serverGroup');
 
         const handleResult = function(command) {
-          var baselineCluster = awsServerGroupTransformer.convertServerGroupCommandToDeployConfiguration(command),
-            canaryCluster = _.cloneDeep(baselineCluster);
+          const baselineCluster = awsServerGroupTransformer.convertServerGroupCommandToDeployConfiguration(command);
+          const canaryCluster = _.cloneDeep(baselineCluster);
```

Comment:
best part of this pr

Reply:
`const`-ing things brings joy to my life.